cookbooks/tile/recipes/default.rb

```diff
@@ -162,6 +162,7 @@
   python3-pyotp
   python3-pyproj
   python3-setuptools
+  python3-pil
```

Comment:
Can we keep this list alphabetically ordered please?

Reply:
done